scripts/issue/284/update_rf_rari_pool.py

```diff
@@ -0,0 +1,29 @@
+from great_ape_safe import GreatApeSafe
+from helpers.addresses import registry
+
+TARGET_RF = 0
+TARGET_MARKETS = [
+    "fUSDC-22",
+    "fFEI-22",
+    "fDAI-22",
+    "fFRAX-22",
+    "fDOLA-22",
+    "fWBTC-22",
+    "fETH-22",
+]
+
+
+def main():
+    dev_msig = GreatApeSafe(registry.eth.badger_wallets.dev_multisig)
+
+    dev_msig.init_rari()
+
+    for key, value in registry.eth.rari.items():
+        if key in TARGET_MARKETS:
+            # check rf
+            current_rf = dev_msig.rari.ftoken_get_rf(value)
+            print(f"Current RF for {key} is {current_rf}, updating it to {TARGET_RF}\n")
+            # update rf
+            dev_msig.rari.ftoken_set_rf(value, TARGET_RF)
```

Comment:
brownie seems not to be able to handle the partial revert along the flow of the transaction and emits the following event:

```
...
Failure
│       ├── error: 10
│       ├── info: 91
│       └── detail: 0
...

which seems to translate into an issue accruing interest. [ref](https://docs.rari.capital/fuse/#error-codes)

while doing a single fToken rf update [sim](https://dashboard.tenderly.co/petrovska/project/simulator/a8096007-29e5-48bc-98cf-7d6de5b5d578) in tenderly seems to exec successfully given the events and state variables changed

Reply:
i propose to ask rari (1) to fix this bug in their system, and (2) ask them to make the necessary parameter changes to our pool until this is fixed.

doing it ourselves without having the ability to properly run, test, review and verify the txs on fork is a big risk imo, one that we shouldnt take if we can prevent it.

even all of their production txs are partially failing because of this, which i am not comfortable with:
<img width="607" alt="Screenshot 2022-04-14 at 12 10 26" src="https://user-images.githubusercontent.com/2835259/163364183-c95c07a3-e9f9-45e2-85d5-c7c1288da02c.png">